app/assets/javascripts/admin.js

```diff
@@ -1,3 +1,23 @@
-// Place all the behaviors and hooks related to the matching controller here.
-// All this logic will automatically be available in application.js.
-// You can use CoffeeScript in this file: http://coffeescript.org/
+$(function() {
+  var allGuests = $('.admin-guest');
+
+  function filterAttending(e) {
```

Comment:
Yeah...will need to figure out how to get angular into the rails asset pipeline though because right now the angular app is in ./public...will spend sometime to figure it out today.